### ISSUE TITLE
Autoboot .cn .hk Cards Disables Bootstrap Menu

### ISSUE THREAD

**Issue by DroidlyMobile** [DS-Homebrew/TWiLightMenu#2439]
**Flashcard or Exploits?** 
**Base Console:** 

**TWiLight Menu++ Version:** 
**TWiLight Menu++ Theme:** 

### Steps to reproduce

1.Chose autoboot for .cn/.hk kernel Ace3DS+, Ace3DS X, R4iLS, r4isdhc.com.cn cards, r4isdhc.hk 2021, R4infinity 2, R4i-XDS 2014 white version
2.Loaded game waited for bootstrap to do its thing
3.Tried my DS/DS Lite/DSi XL all devices no matter what don't enable the in game L/DOWN/SELECT menu
4.Removed autoboot using default .cn/.hk kernel
5.Loaded TWilight Menu manually via the defauly kernel
6.Loaded a game, the menu shows when loading TWilight Menu without the autoboot
7.I also tried multiple SD cards thinking maybe it had something to do with the card but still same issue please check the Autoboot for this card as it worked a while ago not sure what changed.

**Comment by SirPrimalform** [comment]
Yup, I'm having the same issue on an r4isdhc.hk 2025 R4 clone. I'm on TWLmenu 27.12.6 and nds-bootstrap 2.4.1.

**Comment by RocketRobz** [comment]
Fixed by https://github.com/DS-Homebrew/nds-bootstrap/commit/a722437e82d0bf572765dc3e92a4dd5cd3c2e6a6
